native/packages/react-native-bpk-component-animate-height/src/BpkAnimateHeight.js

```diff
@@ -102,13 +128,16 @@ class BpkAnimateHeight extends React.Component {
           ref={ref => {
             this.innerViewRef = ref;
           }}
-          style={innerStyle}
+          style={[innerStyle, STYLES.measure]}
```

Comment:
So we render this `View` soley for the side effect of getting a measure? Can we write a comment somewhere explaining this?

Reply:
yeah, after the upgrade the inner view does not render outside its parent height on IOS (unless position is absolute). We need this extra view just to measure the child's height. I've tried with a single child using position absolute but it doesn't behave the same in some cases, especially style wise, if I make the "visible" view absolute it may affect the visual of its childrens.